CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change log
 
+## [v0.5.1] - 2017-07-24
```

Comment:
yea i meant 5.0.0... sorry about that.  Should I put Unreleased for now or leave CHANGLOG unchanged as part of this PR?